src/codeu/chat/client/simplegui/ChatGuiFX.java

```diff
@@ -21,19 +20,18 @@
     private final static Logger.Log LOG = Logger.newLog(ChatGuiFX.class);
 
```

Comment:
nit: extra white space. There are a few places that have this so I won't flag them all.